op-deployer/pkg/deployer/state/state.go

```diff
@@ -96,6 +113,8 @@ type ChainState struct {
 	DelayedWETHPermissionedGameProxyAddress   common.Address `json:"delayedWETHPermissionedGameProxyAddress"`
 	DelayedWETHPermissionlessGameProxyAddress common.Address `json:"delayedWETHPermissionlessGameProxyAddress"`
 
+	Artifacts ArtifactFilepaths `json:"artifactFilepaths"`
```

Comment:
Does it work to only define these fields in the `State` and not the `Intent` struct?

Reply:
These shouldn't be defined in state/intent at all. Will clarify in a separate comment.